ui-tests/src/test/resources/features/integrations/template.feature

```diff
@@ -0,0 +1,309 @@
+    Then set the template type to "Velocity"
+    Then input template "$firstname $surname works at $company"
+    Then click on the "Done" button
+
+      # add data mapper step for template
+    Then check visibility of page "Add to Integration"
+    When add integration step on position "1"
+    And select "Data Mapper" integration step
+    Then check visibility of data mapper ui
+    Then create data mapper mappings
+      | first_name | firstname |
+      | last_name  | surname   | 
+      | company     | company   |
+    And scroll "top" "right"
+    And click on the "Done" button
+
+    #add mapping for email step
+    Then check visibility of page "Add to Integration"
+    When add integration step on position "3"
+    And select "Data Mapper" integration step
+    Then check visibility of data mapper ui
+    Then open data bucket "4 - Template JSON Schema"
+    Then create data mapper mappings
+      | message      | text    |
+    And scroll "top" "right"
+    And click on the "Done" button
+
+    # finish and save integration
+    When click on the "Save" button
+    And set integration name "DB to gmail (template)"
+    And publish integration
+
+    Then Integration "DB to gmail (template)" is present in integrations list
+    # wait for integration to get in active state
+    And wait until integration "DB to gmail (template)" gets into "Running" state
+    
+    #Wait for integration to really be running
+    And sleep for "10000" ms
+
+    Then check that email from "jbossqa.fuse@gmail.com" with subject "syndesis-template-test" and text "Joe Jackson works at Red Hat" exists
+
+  @db-template-freemarker-send
+  Scenario: Create integration from twitter to gmail with email text formatted by template (freemarker)
+    # create integration
+    When navigate to the "Home" page
+    And click on the "Create Integration" button to create a new integration.
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+      # select twitter connection as 'from' point
+    When select the "PostgresDB" connection
+    And select "Periodic SQL invocation" integration action
+    Then fill in invoke query input with "SELECT * FROM contact" value
+    And click on the "Done" button
+    Then check that position of connection to fill is "Finish"
+
+    # select postgresDB connection as 'to' point
+    Then check visibility of page "Choose a Finish Connection"
+    When select the "QE Google Mail" connection
+    And select "Send Email" integration action
+    Then fill in values
+      | Email to | jbossqa.fuse@gmail.com |
+      | Email subject | syndesis-template-test |
+    And click on the "Done" button
+
+    #adding mustache template to format the mail
+    Then check visibility of page "Add to Integration"
+    When add integration step on position "0"
+    Then select "Split" integration step
+
+    #adding mustache template to format the mail
```

Comment:
Same question as above.

Reply:
@mmuzikar what to do with this? it is just the comment, but anyway :-)